File: backend/src/crowntest/exceptions/VerificationFailedException.ts
export class VerificationFailedException extends Error {
  constructor(message: string, cause?: Error) {
    super(message);
    this.name = 'VerificationFailedException';
    if (cause) {
      this.stack += `\nCaused by: ${cause.stack}`;
    }
  }
}

File: crowntest-sdk/src/main/java/com/crowntest/sdk/exceptions/VerificationFailedException.java
package com.crowntest.sdk.exceptions;

public class VerificationFailedException extends AssertionError {

    public VerificationFailedException(String message) {
        super(message);
    }

    public VerificationFailedException(String message, Throwable cause) {
        super(message, cause);
    }
}
